curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -53,6 +56,19 @@ DEFINE_validator(avgReadFileBytes, &pass_uint64);
 DEFINE_uint64(avgReadFileIops, 0, "the read throttle iops of disk cache");
 DEFINE_validator(avgReadFileIops, &pass_uint64);
 
+DEFINE_uint32(diskNearFullRatio, 70, "the nearfull ratio of disk cache");
+DEFINE_validator(diskNearFullRatio, &pass_uint32);
+DEFINE_uint32(diskFullRatio, 90, "the nearfull ratio of disk cache");
```

Comment:
may be percent is more suitable over ratio

Reply:
> may be percent is more suitable over ratio

This is the name in the configuration file. If you want to change it, you feel that the configuration file must also be changed, which will destroy part of the compatibility this time.But I think you are right, see if you want to change